scripts/update/Updater.php

```diff
@@ -353,5 +354,12 @@ public function update($initialVersion)
 
         $this->skip('5.12.0', '5.12.1');
 
+        if ($this->isVersion('5.12.1')) {
+            $service = new StateService;
+            $service->setServiceManager($this->getServiceManager());
+            $this->getServiceManager()->register(StateService::SERVICE_ID, $service);
```

Comment:
if you add ```getDeliveryStates``` to the current proctoring state service you don't have to register a new service